VERSION

```diff
@@ -1,2 +1,2 @@
 # should contain the currently released version of the formula
-1.0.0
+0.1.0
```

Comment:
`0.1.1` to match the value in [`CHANGELOG.rst`](https://github.com/saltstack-formulas/template-formula/pull/20/files#diff-de0a9b67ffe2874a076b5fdb15bad484R4).

Reply:
yeah but I need to have a commit hash to make a tag.. I think........

Reply:
But when this is merged, it will _be_ `v0.1.1`.